rdr_client/work_queue.py

```diff
@@ -3,16 +3,25 @@
 
 import logging
 import pprint
+
 from main_util import configure_logging
 from client import Client
 
 
 def main():
   client = Client()
+  AWARDEE = 'PITT'
```

Comment:
We may want to add comments in here:

- the purpose of this code -- syncing participant summaries from the RDR
- you should call code like this with credentials for a service account that has been set up for your awardee by AllOfUs
- you expect to get the same participant summaries back in multiple responses (anything modified within 60 seconds of the last summary in the previous response)
- you'll get back relation="next" on the link if there are more results available in the RDR that weren't returned vs. relation="sync" if there were. (Though I think we probably want them to sync every five minutes either way.)
- provide a link to the README describing the participant summary fields and the API generally

Then this code can serve as an example to HPOs for how to integrate with the API.

Reply:
done